charts/kuberpult/run-kind.sh

```diff
@@ -294,8 +294,8 @@ kubectl get pods
 
 if "$LOCAL_EXECUTION"
 then
-  echo "sleeping for 1h to allow debugging"
-  sleep 1h
+  echo "sleeping to allow debugging. press any key to exit"
```

Comment:
```suggestion
  echo "sleeping to allow debugging. press enter to exit"
```
pretty sure `read` only reacts on enter or ctrl-c